activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/service/api/ProjectService.java

```diff
@@ -59,11 +59,11 @@ Project copyProject(Project projectToCopy,
 
     Project replaceProjectContentWithProvidedModelsInFile(Project project, InputStream inputStream) throws IOException;
 
-    default Page<Project> getProjects(Pageable pageable, String name, List<String> filters) {
+    default Page<Project> getProjects(Pageable pageable, String name, List<String> filters, List<String> include) {
```

Comment:
why we are passing the value and then we are not using it?

Reply:
I will use it in enterprise repo where we implement this service. This way I won't need to change the implementation  in activiti-cloud.